lncfg/protocol_experimental_off.go

```diff
@@ -1,25 +0,0 @@
-//go:build !dev
-// +build !dev
-
-package lncfg
-
-import (
-	"github.com/lightningnetwork/lnd/feature"
-	"github.com/lightningnetwork/lnd/lnwire"
-)
-
-// ExperimentalProtocol is a sub-config that houses any experimental protocol
-// features that also require a build-tag to activate.
-type ExperimentalProtocol struct {
```

Comment:
i still think we want to leave this struct here. Ie, leave the framework in place & just move the custom fields out

Reply:
if you "git blame", you can see that the framework was added independently of the custom message options

Reply:
Thanks for taking a look @ellemouton! That makes sense. I reworked to leave that framework in place and just move the "custom" protocol features out. Lmk if this is what you had in mind.